ega-data-api-dataedge/src/main/resources/application.yml

```diff
@@ -1,15 +1,29 @@
-spring.profiles: no-oss
+spring.profiles.active: no-oss,logger-log,add-user-ip-headers
 
 eureka.client.enabled: false
 
+management.server.port: -1
+endpoints.enabled: false
+endpoints.health.enabled: true
+endpoints.info.enabled: true
+endpoints.refresh.enabled: false
+
 # Used when a custom token is generated and we have an ASCII encoded key
 security.oauth2.resource.jwt.key-value: ${JWTKEY}
 
+
 FILEDATABASE:
-  listOfServers: ${FILEDATABASE_SERVERS:filedatabase}
+  listOfServers: ${FILEDATABASE_SERVERS:localhost:8080}
```

Comment:
unintended change?

Reply:
will revert, I think the logic was that one would start this on `localhost` for testing, but does not make sense in retrospective.